torch/_inductor/utils.py

```diff
@@ -1812,9 +1812,15 @@ def tensor_is_aligned(tensor: torch.Tensor):
     # but symbolic storage_offsets are. For consistency, we suppress guard creation
     # upon performing this check: that ensures that we don't add recompiles when we
     # add this logic.
-    return (
-        tensor.storage_offset() * get_dtype_size(tensor.dtype)
-    ) % GPU_ALIGN_BYTES == 0
+    from torch.fx.experimental.symbolic_shapes import guard_size_oblivious
+
+    try:
+        return guard_size_oblivious(
```

Comment:
Wonder whether we need `guard_size_oblivious` here. From my understanding, it can not prevent the exception because we don't know whether `storage_offset()` returns a even number of not

Reply:
Yeah, size oblivious is wrong here. It probably doesn't matter, because all size oblivious will do is make us assume storage_offset isn't zero or one, which won't affect whether or not we can determine if the modulus is zero. But better to drop it.